src/collective/collectionfilter/utils.py

```diff
@@ -16,12 +17,18 @@ def target_collection_base_path(context):
     return '/'.join(context.getPhysicalPath())
 
 
+def target_collection_types(context):
```

Comment:
may you remove this function as it's not used in `2.x` branch? it can be backportet in another PR in my opinion but it's unrelated to this PR